generalized_linear_regression/generalized_linear_regression.md

```diff
@@ -0,0 +1,408 @@
+**Generalized linear models are a way to extend the functionality of linear models to cover situations when they normally wouldn't work.**
+
+## Why do we need generalized linear models?
+
+Linear models are only appropriate when your outcome variable(s) are **continuous** and **unbounded**. 
+If you want to model a categorical outcome, or one that's bounded (like a proportion or percent), then usually can't rely on regular linear modeling to get the job done. 
+
+<h3>What counts and continuous and unbounded?</h3>
+
+A variable is **continuous** if it can take any value on a scale, not just discrete options. 
+For example, age is generally continuous because you can be 11 years old, or 11.5, or 11.476 years old, etc.
+In contrast, experimental group (treatment vs. control) is not continuous; there's no way to be halfway between treatment and control, you're in one or the other. 
+Something like medication dose could be continuous or categorical depending on how it's used. 
+If patients are restricted to categorical levels of dose (e.g. high, med, or low dose groups) then it would be categorical. 
+But if it's something where their dose might theoretically be anywhere on the scale, then it's continuous.
+
+A variable is **unbounded** if it can theoretically extend from negative infinity at the low end to positive infinity at the high end. 
+We often have variables that aren't truly unbounded, but they're close enough for all practical purposes -- for example, IQ can't technically be negative, but in practice no measured IQs are close to 0 so the data stop naturally before they hit a the scale boundary at 0. 
+IQ is generally treated as unbounded statistically.
+
+Something like a probability or proportion is bounded at 0 and 1; it's not possible to have a negative probability, or one over 1. 
+
+A variable like age might be unbounded or bounded depending on what ages you're studying. 
+In a young pediatric population, age is bounded at 0 because you can't have any negative ages and a distribution of infant ages will get cutoff at that lower bound. 
+But if you're studying adults, then age might be functionally unbounded in your dataset.
+
+<div class = "learn-more">
+<b style="color: rgb(var(--color-highlight));">Learning connection</b><br>
+
+For an excellent review of continuous vs. categorical variables, see the [Khan Academy video on discrete and continuous random variables](https://www.khanacademy.org/math/statistics-probability/random-variables-stats-library/random-variables-discrete/v/discrete-and-continuous-random-variables).
+
+</div>
+
+We can coerce problematic outcome variables into nice, continuous ones so that we can still model them with a line.
+The strategy for transforming your outcome depends on what its distribution is like to begin with --- if it's binary (or bounded at 0 and 1), then a logit works well. 
+If it's count data, then a log is the right transformation, etc.
```

Comment:
I'd suggest moving some of the technical content of this paragraph later and focus on the first sentence. Explain what it means to "coerce problematic outcome variables" without introducing log and logit yet.

In fact I think I'd move the Logistic Regression intro section up here, maybe with structure like:

Why do we need generalized linear models? (linear only works for unbounded, continuous)
- We can transform/make/coerce bounded or continuous data into a new shape so that linear stuff works again!
- log and logit are the way to do this
- more details for the mathematically interested, which is your current big behind the scenes box.

Reply:
Okie dokie, @drelliche  I reordered things, as per our discussion -- take a look and let me know how you think it's landing now! 
I also tried to tidy up the mathjax, but please be picky and let me know if it's still not great!